packages/nx-plugin/src/executors/playwright/schema.d.ts

```diff
@@ -0,0 +1,5 @@
+import { PlaywrightExecutorSchema as NxPlaywrightExecutorSchema } from '@nx/playwright';
+
+export interface PlaywrightExecutorSchema extends NxPlaywrightExecutorSchema {
+  testingType?: 'component';
```

Comment:
following cypress executor official plugin APIs 

```suggestion
/**
   * @default 'component'
   */
  testingType?: 'component' | 'e2e';
```